cmd/deploy.go

```diff
@@ -77,6 +112,32 @@ func (c DeployCmd) Run(opts DeployOpts) error {
 	return c.deployment.Update(bytes, updateOpts)
 }
 
+func setFlags(flags []string, opts DeployOpts) DeployOpts {
+	for j := range flags {
+		switch flags[j] {
+		case "fix-releases":
+			opts.FixReleases = true
+		case "fix":
+			opts.Fix = true
+		case "recreate":
+			opts.Recreate = true
+		case "recreate-persistent-disks":
+			opts.RecreatePersistentDisks = true
+		}
+	}
+
+	return opts
+}
+
+func containsDeployment(slice []string, value string) bool {
```

Comment:
Thinking of the [implementation done in Director for addons](https://github.com/cloudfoundry/bosh/blob/main/src/bosh-director/lib/bosh/director/addon/addon.rb#L39-L42), I think that `applies()` would be better name for this function, with this pseudo code as implementation:
```
list == null OR list contains deploymentName
```